.circleci/config.yml

```diff
@@ -31,15 +31,37 @@ jobs:
       - run: composer require silverstripe/recipe-cms
       - run: composer update -n --prefer-dist --ignore-platform-reqs
       - run: composer vendor-expose
+      - run:
+          name: Setup Integration Pre-requisites
+          command: |
+            if [ ! -d assets ]; then mkdir assets; fi
+            cp .circleci/.env.test ./.env
+            cp tests/mocks/TestIndex.php app/src/CircleCITestIndex.php
+            sed -i -e 's/namespace Firesphere\\SolrSearch\\Tests;//' app/src/CircleCITestIndex.php
+            sed -i -e "s/ implements TestOnly//" app/src/CircleCITestIndex.php
+            sed -i -e "s/TestIndex/CircleCITestIndex/" app/src/CircleCITestIndex.php
```

Comment:
Could you make this a separate class to be copied from the `.circleci` folder instead? It would be a lot clearer to read :)